src/api/spec/factories/comment.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :comment do
+    body { Faker::Lorem.paragraph }
+    type { Faker::Lorem.word }
```

Comment:
You shouldn't use a random word as a `type` as it is reserved for storing the the inheritance class. Because of the same reason and taking into account that it is not possible to create a `comment `without type (because of the presence validation) it doesn't make sense to test `comment.` You should test the comment children. I think that for some function it is enough with testing it for one child (for example comment_package), although for some of them it may be needed to test it for more.

Also, I added a validation for that in https://github.com/openSUSE/open-build-service/pull/2252 and this should fail in the tests after it is merged (it already fails outside the tests).